### PR TITLE
Reproducing ROC curves

### LINKED ISSUE #1: Duplicate keys in attack parameter dictionaries
Hi,

some of the attack parameter dictionaries contain the key `"eps_iter"` two times. The dictionaries with

-  `"method": "bim"`, `"eps": 5` and
-  `"method": "mim"`, `"eps": 5` do so.

Cheers,
Johannes

https://github.com/lsgos/uncertainty-adversarial-paper/blob/559af3e4a5df7727db3ec6c3493831c81618887c/ROC_curves_cats.py#L359-L394

### LINKED ISSUE #2: Missing parameters to reproduce ROC curves
Hi, 
thank you for making the code needed to reproduce the experiments from your paper publicly available. Currently, I'm trying to reproduce the ROC curves in figure 14 from the paper. Apparently the default parameters of this code version produce very different results, some examples attached below.

Could you please share the necessary parameter configuration for

- number of examples
- number of MC passes
- exact configuration for the attack parameters
- seed for the random number generator to select the same images

Training the model worked fine. The validation accuracy after 40 epochs was about 94%. However, it would also be nice if you could share your model parameters to rule out any doubts in this regard.

I just added a title to the plot to identify the related attack configuration. I haven't changed any of the code's logic.
image
image

### PR DESCRIPTION
**Pull request by johannesjung**
I want to resolve #2 (and fix #1) and add some additional instructions on how to reproduce the ROC curves of the paper's figure 14.

In contrast to the previous default settings, I was now able to produce very similar ROC curves with the discussion on #2 :

my_roc_curves_fgm_6roc_curves
my_roc_curves_fgm_7roc_curves
my_roc_curves_bim_6roc_curves
my_roc_curves_bim_8roc_curves
my_roc_curves_mim_1roc_curves
my_roc_curves_mim_2roc_curves

### CODE REVIEW

**Comment by lsgos** [comment]
Thanks again for trying to fix this, and I'm sorry you had to chase me to get the *actual* parameters I used.
